In [37]:
from pymongo import MongoClient
client = MongoClient()
db = client.whosampled

import pandas as pd

df = pd.DataFrame(list(db.main_redo.find()))

import itertools

In [39]:
df.shape

(15809, 16)

There are 15809 unique records. There are 1324 producers. There are 7789 songs sampled.

In [96]:
utility2.head(50)

It's a New Day  We Want to Parrty, Parrty, Parrty  \
"Buttnaked" Tim Dawg                     1                                  0   
$uicideboy$                              0                                  0   
10                                       0                                  0   
12 Million                               0                                  0   
1500 or Nothin'                          0                                  0   
2 in Rhythm                              0                                  0   
2Pac                                     0                                  0   
3 Boyz From Newark                       1                                  0   
30 Roc                                   0                                  0   
42 North                                 0                                  0   
45 Music                                 0                                  0   
4th Disciple                             1                                  0   
5th Ward Boyz                            0                                  0   
6ix                                      0                                  0   
7 Aurelius                               0                                  0   
7L                                       0                                  0   
8-Off Agallah                            0                                  0   
808 Mafia                                0                                  0   
808&amp;EliTE                            0                                  0   
808-Ray                                  0                                  0   
88-Keys                                  0                                  0   
8Track                                   0                                  0   
9th Wonder                               0                                  0   
A Kid Called Roots                       0                                  0   
A New Beginning                          0                                  0   
A Tribe Called Quest                     0                                  0   
A$AP Ty Beats                            0                                  0   
A-Trak                                   0                                  0   
A. G. Cook                               0                                  0   
AHk 2Gs                                  0                                  0   
Aalias                                   0                                  0   
Abraham Orellana                         0                                  0   
Adam Bravin                              0                                  0   
Adam Deitch                              0                                  0   
Adam Pallin                              0                                  0   
Adrian Younge                            0                                  0   
Aesop Rock                               0                                  0   
Afrika Islam                             0                                  0   
Agallah                                  0                                  0   
Ahmir "Questlove" Thompson               0                                  0   
Air Smooth                               0                                  0   
Aja Grant                                0                                  0   
Akhenaton                                0                                  0   
Akon                                     0                                  0   
Al "Butter" McLean                       0                                  0   
Al Be Back                               0                                  0   
Al West                                  0                                  0   
Alan Braxe                               0                                  0   
Alex Coletti                             0                 

In [102]:
utility2.sum(axis = 1)

"Buttnaked" Tim Dawg      2
$uicideboy$               6
10                        1
12 Million                1
1500 or Nothin'           1
2 in Rhythm               2
2Pac                      3
3 Boyz From Newark        8
30 Roc                    4
42 North                  1
45 Music                  1
4th Disciple            105
5th Ward Boyz             1
6ix                       8
7 Aurelius               18
7L                        2
8-Off Agallah             1
808 Mafia                 9
808&amp;EliTE             4
808-Ray                   5
88-Keys                  52
8Track                    1
9th Wonder              983
A Kid Called Roots        3
A New Beginning           5
A Tribe Called Quest      1
A$AP Ty Beats             3
A-Trak                    7
A. G. Cook                1
AHk 2Gs                   1
                       ... 
Wally West                1
Warryn Campbell           1
Watts                     2
WestBam                   4
Wheezy 5th          

In [105]:
sum(utility2.loc['Alex Coletti'])

3